### ISSUE TITLE
[Admin Dashboard] Heading order does not reflect data structure

### ISSUE THREAD

**Issue by johanna-skylight** [CDCgov/prime-simplereport#5810]
## Area or Component
Admin Dashboard all four summary cards.

## Quick Summary

The headings placed in the summary cards do not reflect the data hierarchy.

### How to Replicate
Inspect the DOM

## Accessibility Violation

This violations does not map 1:1 to the issue I am reporting but was the closest to the problem. The heading does not describe the topic or purpose of the content that follows it.
https://docs.deque.com/issue-help/1.0.0/en/heading-not-descriptive/

## Action Requested

Set the Heading level in the cards to H3.

## Screenshots (optional)
![Screenshot 2023-05-15 at 4 40 35 PM](https://github.com/CDCgov/prime-simplereport/assets/103958711/fdb6f105-06d0-4300-9ae5-998e7b29adc0)
